src/instruments/src/EFB/Ground/Ground.tsx

```diff
@@ -31,7 +32,7 @@ export const Ground = ({
     const [pushBack, setPushBack] = useSplitSimVar('PUSHBACK STATE', 'enum', 'K:TOGGLE_PUSHBACK', 'bool', 1000);
     const [powerActive, setPowerActive] = useSplitSimVar('A:INTERACTIVE POINT OPEN:8', 'Percent over 100', 'K:REQUEST_POWER_SUPPLY', 'bool', 1000);
 
-    const [pushBackWait, setPushBackWait] = useSimVar('Pushback Wait', 'bool', 100);
+    const setPushBackWait = useSimVarSetter('Pushback Wait', 'bool');
```

Comment:
Could we just keep using `useSimvar` to avoid another import? I believe you could do something like this `const [, setPushBackWait] = useSimVar('Pushback Wait', 'bool');`

Reply:
Will do.